react-backend/client/src/App.js

```diff
@@ -56,154 +57,156 @@ class App extends Component {
-            path="/visitor/signup/*"
-            component={Main}
-          />
-
-          <PrivateRoute
-            auth={this.state.loggedIn}
-            exact
-            path="/visitor/login"
-            component={QRCode}
-          />
-
-          <PrivateRoute
-            auth={this.state.loggedIn}
-            exact
-            path="/visitor/qrerror"
-            component={QrError}
-          />
-
-          <PrivateRoute
-            auth={this.state.loggedIn}
-            exact
-            path="/visitor/end"
-            component={redirectAfterTimeout('/visitor', 5000)(Thanks)}
-          />
-
-          <PrivateRoute
-            auth={this.state.loggedIn}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/login"
-            component={ConfirmPassword}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin"
-            updateLoggedIn={this.updateLoggedIn}
-            component={AdminMenuPage}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/activities"
-            component={AdminActivitiesPage}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/visits"
-            updateLoggedIn={this.updateLoggedIn}
-            component={AdminVisitsPage}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/users"
-            updateLoggedIn={this.updateLoggedIn}
-            component={AdminUsersPage}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/user/:userId"
-            updateLoggedIn={this.updateLoggedIn}
-            component={AdminUserDetailsPage}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/accountSettings"
-            updateLoggedIn={this.updateLoggedIn}
-            component={AdminCBSettingsPage}
-          />
-
-          <AdminRoute
-            auth={this.state.adminToken}
-            updateAdminToken={this.updateAdminToken}
-            exact
-            path="/admin/feedback"
-            updateLoggedIn={this.updateLoggedIn}
-            component={adminLoginCheck(CbAdminFeedbackPage)}
-          />
-
-          <Route exact path="/internalServerError" component={InternalServerError} />
-          <Route component={NotFound} />
-        </Switch>
-      </Container>
+      <Dots>
```

Comment:
How come there's such a big dif here? Can this be avoided as this caused me some major rebase woes last time.

Reply:
An extra wrapping element means the indentation level for all children increases by one.

Reply:
What changes did you make to the App.js? Just swap out some components in the route definitions? If so, handling conflicts on rebase is simple: accept the current changes (i.e. changes _not_ coming from your commits), then past your components back in.